src/node/chainstate.cpp

```diff
@@ -28,11 +28,8 @@ std::optional<ChainstateLoadingError> LoadChainstate(bool fReset,
     };
 
     LOCK(cs_main);
-    chainman.InitializeChainstate(mempool);
-    chainman.m_total_coinstip_cache = nCoinCacheUsage;
-    chainman.m_total_coinsdb_cache = nCoinDBCache;
```

Comment:
Why are `m_total_coinstip_cache` and `m_total_coinsdb_cache` not initialized anymore? As it stands, bitcoind compiles with both those members marked as `static constexpr` which doesn't seem right...